colorbleed/maya/lib.py

```diff
@@ -1292,3 +1312,68 @@ def _get_parent(node):
         _id = get_id(similar_node)
         if _id:
             return _id
+
+
+def set_project_fps():
+    """Set FPS from project configuration
+
+    Returns:
+        None
+
+    """
+
+    int_fps = [15, 24, 5, 30, 48, 50, 60, 44100, 48000]
```

Comment:
Can we make these sets instead of list for optimal lookup (faster)

```python
int_fps = {15, 24, 5, 30, 48, 50, 60, 44100, 48000}
float_fps = {23.976, 29.97, 29.97, 47.952, 59.94}
```